docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -0,0 +1,60 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    private static readonly Meter MyMeter = new Meter("MyCompany.MyProduct.MyLibrary", "1.0");
+
+    public static void Main(string[] args)
+    {
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddSource(MyMeter.Name)
+            .AddView(instrumentName: "MyCounter") // MyCounter will be reported with defaults. This is done to ensure that any other wildcard Views for the same isntrument does not affect this instrument.
```

Comment:
@mic-max @Yun-Ting wish to get your feedback on these regarding the usability / easy-to-understand part.

Reply:
my gut says, unless one understands View's concept from the spec (and concept of metric stream), this is not super intuitive. :)

Reply:
I see you have a class for `MetricStreamConfig`. How about a class for `InstrumentSelectionCriteria` as well?

So the signature would be `AddView(SelectionCriteria, MetricStreamConfig)`.

Later we might consider some builder methods for these classes as well. Something like:

```csharp
var selectionCriteria = SelectionCriteria
    .Create()
    .MeterName("SomeMeter")
    .InstrumentName("SomeInstrument");

var viewConfig = MetricStreamConfig
    .Create()
    .Name("NewNameForSomeMeter")
    .Aggregation(Aggregation.Histogram);
```

Reply:
MetricStreamConfig is internal (my bad missed removing it from public!).
User doesn't see that.